platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/Browser.java

```diff
@@ -268,6 +266,11 @@ private static void launchUnhandled(@NonNull final CommandLineArgs commandLineAr
             appMains.get(nodeId).init(platforms.get(nodeId), nodeId);
         }
 
+        if (commandLineArgs.pcesRecovery()) {
+            platforms.values().forEach(SwirldsPlatform::performPcesRecovery);
+            SystemExitUtils.exitSystem(SystemExitCode.NO_ERROR, "PCES recovery done");
+        }
```

Comment:
It seems strange to me that we would ever try to recover multiple nodes simultaneously.

Reply:
I agree, but I would have the make a lot more changes to make it single platform. This all comes down to our startup, once we clean this up, we can make better changes.

Reply:
done